Mathlib/Combinatorics/SetFamily/KruskalKatona.lean

```diff
@@ -282,7 +282,7 @@ theorem kruskal_katona (h𝒜r : (𝒜 : Set (Finset (Fin n))).Sized r) (h𝒞
 /-- An iterated form of the Kruskal-Katona theorem. In particular, the minimum possible iterated
 shadow size is attained by initial segments. -/
 theorem iterated_kk (h₁ : (𝒜 : Set (Finset (Fin n))).Sized r) (h₂ : #𝒞 ≤ #𝒜) (h₃ : IsInitSeg 𝒞 r) :
-    #(∂^[k] 𝒞) ≤ #(∂^[k] 𝒜) := by
+    #(∂ ^[k] 𝒞) ≤ #(∂ ^[k] 𝒜) := by
```

Comment:
I'm not sure if this is how the author meant it. @YaelDillies Do you know (or know the author and can ask them)?